Sources/TSCBasic/Process.swift

```diff
@@ -230,6 +230,8 @@ public final class Process {
     // process execution mutable state
     private var state: State = .idle
     private let stateLock = Lock()
+    private static let sharedCompletionQueue = DispatchQueue(label: "Process.sharedCompletionQueue")
```

Comment:
Would be nice to use `org.swift.tools-support-core.process-completion` as the name for the queue.

Reply:
Fixed in 95e286b.